roles/openshift_node/tasks/main.yml

```diff
@@ -94,6 +94,13 @@
     - openshift.common.use_openshift_sdn | default(true) | bool
     - not openshift.common.is_containerized | bool
 
+- name: Restart cri-o
+  systemd:
+    name: cri-o
+    enabled: yes
+    state: restarted
+  when: openshift_docker_use_crio | default(false)
```

Comment:
We could. The reason I put it here though was that in my testing I always did it after the sdn was installed as that was what we needed more so than the node doing anything.